docs/reference/rest-api/rest-api-compatibility.asciidoc

```diff
@@ -32,20 +27,18 @@ IMPORTANT: REST API compatibility does not guarantee the same behavior
 as the prior version. It instructs {es} to automatically resolve any
 incompatibilities so the request can be processed instead of returning an error.
 
-
 REST API compatibility should be a bridge to smooth out the upgrade process,
 not a long term strategy. REST API compatibility is only honored across one
-major version: honor 7.x requests/responses from 8.x.
+major version: honor 8.x requests/responses from 9.x.
 
 When you submit requests using REST API compatibility and {es} resolves
 the incompatibility, a message is written to the deprecation log with
 the category "compatible_api". Review the deprecation log to identify
 any gaps in usage and fully supported features.
 
-
-For information about specific breaking changes and the impact of requesting
-compatibility mode, see <<breaking_80_rest_api_changes, REST API changes>>
-in the migration guide.
+//TODO: figure out how to get the link below to work !
+//For information about specific breaking changes and the impact of requesting
+//compatibility mode. See xref:/docs/reference/migration/migrate_9_0/rest-api-changes.asciidoc[REST API changes] in the migration guide.
```

Comment:
```suggestion
//compatibility mode. See <<rest-api-changes,REST API changes>> in the migration guide.
```

driving by here :)

Reply:
Thanks Liam, I tried that but got:
```
INFO:build_docs:asciidoctor: WARNING: invalid reference: rest-api-changes
```
The link is trying :
docs/reference/rest-api/rest-api-compatibility.asciidoc <- from here
docs/reference/migration/migrate_9_0/rest-api-changes.asciidoc <- to here

This also, does not work: `See <<breaking_90_rest_api_changes,REST API changes>>` 
```
INFO:build_docs:asciidoctor: WARNING: invalid reference: breaking_90_rest_api_changes
```

I only updated the file name and anchor name to be `9` instead of `8`. This used to work : `<<breaking_80_rest_api_changes, REST API changes>>` 

So I assume that maybe I removed some implicit transitive include or something like that (I am pretty clueless with asciidoc)